transports/msmq/distributor/configuration.md

```diff
@@ -76,3 +76,87 @@ If self-hosting the endpoint here is the code required to enlist the endpoint wi
 snippet: ConfiguringWorker
 
 Similar to self-hosting, if running NServiceBus prior to Version 6, ensure the `app.config` of the worker contains the `MasterNodeConfig` section to point to the hostname where the distributor process is running.
+
+## Troubleshooting
+
+### Performance issues, throughput is low
+
+#### Maximum concurrency
+
+Make sure that the maximum concurrency level is increased on the distributor. By default, the maximum concurrency level is 1. Also, make sure that the workers have the maximum concurrency level set higher.
+
+#### Prefetching
+
+When your workers are using NServiceBus 6, then there is the ability to configure the worker capacity independently from the maximum concurrency. By assigning a capacity value higher then the maximum concurrency you are able to reduce the latency to start working on a next message as by default the worker first reports it is available for work and then gets a new message forwarded by the distributor.
+
+#### Sender side distribution
+
+Using [sender side distribution](h/transports/msmq/sender-side-distribution) instead of using the distributor might be a better alternative if you are able to use NServiceBus 6 and is much more suitable for scenarios where IO is the bottleneck. 
```

Comment:
> might be a better alternative

that sounds prescriptive. Doesn't it depend on the pros and cons of each situation? i.e. sending lots of messages to a worker that's unavailable vs only sending ones that check in with the distributor for example?